CHANGELOG.md

```diff
@@ -12,6 +12,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Detect-only Redact for Sanitize.
 - Support for `domains` field in `v2/user/breached` endpoint in User Intel service
 
+### Changed
+
+- Vault APIs have been updated to v2.
+
+### Removed
+
+- Deprecated functions `UserListRequest.Builder.setFilter(Filter filter)` and
```

Comment:
nit: would be nice to indicate which service these classes are from

Reply:
Sure, done.